add-on/panels/test/roomStore_test.js

```diff
@@ -178,6 +179,36 @@ describe("loop.store.RoomStore", function() {
       });
 
       describe("close", function() {
+        it("should request rename if closing room is last created", function() {
+          store.setStoreState({
+            openedRoom: "fakeToken",
+            lastCreatedRoom: "fakeToken"
+          });
+
+          sinon.stub(store, "setStoreState");
+
+          LoopMochaUtils.publish("Rooms:Close");
+
+          sinon.assert.calledTwice(store.setStoreState);
+          sinon.assert.calledWithExactly(store.setStoreState.getCall(0),
+                                         { closingNewRoom: true });
+          sinon.assert.calledOnce(requestStubs.SetNameNewRoom);
+        });
+
+        it("should end up updating  closingNewRoom state to false", function() {
```

Comment:
nit: double space. Though something like `should set closingNewRoom state to false` might be better.